src/memberdef.cpp

```diff
@@ -4549,6 +4548,31 @@ void MemberDefImpl::setInitializer(const QCString &initializer)
   while (p>=0 && isspace(static_cast<uint8_t>(m_initializer.at(p)))) p--;
   m_initializer=m_initializer.left(p+1);
   m_initLines=m_initializer.contains('\n');
+  QCString s(m_initializer);
```

Comment:
Doesn't this do the same as `stripIndentationVerbatim()` in `util.cpp`?

Reply:
The comment with the function says:
> (excluding the first line)

and later on:
> skip the initial line from stripping


So I don't think this is the case, the function could be enhanced in a way that also the first line is considered (e.g by an argument `bool stripFirtstLine = false` that in this case can be set to `true`)

Feel free to adjust this.

Reply:
Used extended `stripIndentationVerbatim` based on review